### PR TITLE
Updating jwks-rsa to v2.0.0

### PR DESCRIPTION
**Pull request by avenmia**
Updating jwks-rsa to v2.0.0 causes the test "Tests for JWKS being correctly consumed by jwks-rsa client › should get the correct key from the jwks endpoint" in jwks.test.ts to fail. 

#36

### CODE REVIEW

**Comment by Levino** [comment]
Do you have an idea how to fix this?

I think it is a valid issue for "keeping things clean reasons". But just wondering why would I want to update the library? In which scenario would this become an issue for the consumer of the library? Should they not just do not care that we use a different version of this lib here? Are you saying "jwks-rsa" is more a peer dependency?

**Comment by avenmia** [comment]
@Levino 
I wasn't able to figure it out. 

I think the issue is in the generation of the certificate because the jose helper library is throwing an exception when validating the x5c property of the certificate. 
https://github.com/Levino/mock-jwks/blob/f89a8986e69ed3ae1de2d57b65b10a609f22d59d/src/tools.ts#L97-L134

In jose/lib/jwk/key/base.js, this is where it's failing. It hits the following error:
'The key in the first `x5c` certificate MUST match the public key represented by the JWK'

image

There's a difference in the certificate made to populate the x5c property and the one they are generating. I originally thought that was the algorithm name and changing RSA256 to RS256 might fix it, but no luck.

I'd expect that most consumers would keep keep their jwks-rsa library up to date for security concerns. I don't understand dependencies well, but it seems jwks-rsa might be more a peer dependency.

**Comment by Levino** [comment]
Please merge `main`. Also use yarn berry please and remove the `package-lock.json` file.

**Comment by avenmia** [comment]
That worked! Thank you!

**Comment by Levino** [comment]
You forgot to run `yarn` to actually fetch and use version 2 of `jwks-rsa`. I fixed that for you in #40. Closing this PR.

File: package.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
     "eslint": "^7.10.0",
     "eslint-config-prettier": "^6.12.0",
     "jest": "^26.5.2",
-    "jwks-rsa": "^2.0.0",
+    "jwks-rsa": "^2.0.1",
```
Comment (Levino) [inline package.json:82]: This change does nothing by the way. I would rather suggest
```
"jwks-rsa": "^2",
```
which is equivalent and would be clearest.

**Pull request closed without merging**